src/qt/guiutil.cpp

```diff
@@ -764,10 +764,10 @@ QString NetworkToQString(Network net)
     assert(false);
 }
 
-QString ConnectionTypeToQString(ConnectionType conn_type)
+QString ConnectionTypeToQString(ConnectionType conn_type, bool relay_txes)
 {
     switch (conn_type) {
-    case ConnectionType::INBOUND: return QObject::tr("Inbound");
+    case ConnectionType::INBOUND: return relay_txes ? QObject::tr("Inbound Full Relay") : QObject::tr("Inbound Block Relay");
```

Comment:
SPV (light-wallet) peers are not "inbound block relay" peers

Reply:
good point, fixing

Reply:
hopefully resolved with #201